tests/test.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import socket
```

Comment:
@thevindu-w I think it is better we have these test cases in a sub folder within tests folder. In near future we should also start the habit of writing unit test cases for JasmineGraph. There we need place to keep those tests as well. WDYT?

Reply:
Moved the integration tests into `tests/integration/`

Reply:
Ok